packages/monitoring/package.json

```diff
@@ -2,10 +2,14 @@
   "name": "@ministryofjustice/hmpps-monitoring",
   "version": "0.0.1",
   "description": "Retrieve and display health and status information from external services and internal components.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/index.cjs.js",
+  "module": "./dist/index.esm.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
   "scripts": {
-    "build": "tsc"
+    "build": "rollup -c rollup.config.js --bundleConfigAsCjs"
```

Comment:
does this do typechecking?

Reply:
> noEmitOnError
Type: Boolean
Default: false
> If a type error is detected, the Rollup build is aborted when this option is set to true

[Do we need to enable this](https://www.npmjs.com/package/@rollup/plugin-typescript#:~:text=noEmitOnError,set%20to%20true.)?

Reply:
probably!